### ISSUE TITLE
accessibility review

### ISSUE THREAD

**Issue by Lane** [Hyperobjekt/covid-19-behind-bars#48]
- [ ] check all UI controls can be used with keyboard
- [ ] check all links / controls have appropriate focus states
- [ ] check screen reader on all pages to ensure elements are read correctly
- [ ] check `alt` attributes on images
- [ ] check proper heading hierarchy on all pages

**Comment by lougroshek** [comment]
## check alt attributes on images

I'm not finding issues with images **except** the states in the national map and the counties in states map, which are each read by screen readers as image, though the spikes are not read at all. States and counties are announced as "image" but have no alt/title/aria-label. 

## check all UI controls can be used with keyboard

**Comment by lougroshek** [comment]
Checking off the `lang` attribute issue as I think that might be a timing issue where react hasn't yet written the attribute when the testing script hits it.

**Comment by Lane** [comment]
@lougroshek thanks for this review and the detail provided! a few notes below

maps:
- short term, maybe we drop the tab index off individual states / counties for now and use an `aria-label` (or `title`) on the SVG as a whole to present it as a static image.  we can update the `title` based on selections and the explainer text around should help communicate and if they want to know specifics they can go to the table.
- a good long term goal might be to have a summary read for each state / county, though i'm not sure this is best.  could be cumbersome for users to navigate through, especially if we're looking at 50+ tabbable elements.

color contrast: could give it a boost, i'll let @sashasashasasha make that call

input labels: we can use the `inputProps` attribute on the material ui `TextField` or `Input` if we want to pass props directly to the input element, but if it is getting picked up by the screen reader as is it may not be a huge priority.

**Comment by lougroshek** [comment]
@Lane Good to know about `inputProps`. Lots to learn with Material UI. 

In the associated PR I've added a `<desc>` to the two map svgs as `<title>` caused a tooltip to be shown on mouseover. `<desc>` is still read by VoiceOver but not displayed on hover. I think in terms of adding aria-labels to each geography, we'd def want that to be conditional since it would only make sense for some applications.
